README.md

```diff
@@ -10,47 +10,99 @@ master|[![Build Status](https://mseng.visualstudio.com/AzureDevOps/_apis/build/s
 Secure platform credential storage|&#10003;<br/>Windows Credential Manager|&#10003;<br/>macOS Keychain|
-Multi-factor authentication support for Azure DevOps|&#10003;|&#10003;|
+Multi-factor authentication support for Azure DevOps|&#10003;|&#10003;|&#10003;\*
 Two-factor authentication support for GitHub|&#10003;|&#10003;\*|&#10003;\*
-Two-factor authentication support for BitBucket|||
+Two-factor authentication support for BitBucket|&#10003;|&#10003;\*|&#10003;\*
 Windows Integrated Authentication (NTLM/Kerberos) support|&#10003;|_N/A_|_N/A_
 Basic HTTP authentication support|&#10003;|&#10003;|&#10003;
+Proxy support|&#10003;|&#10003;|
 
 **Notes:**
 
 (\*) Currently only supported when using Git from the terminal or command line. A platform-native UI experience for GitHub is not yet available on macOS, but planned.
 
 ### Planned features
 
-- [ ] Proxy support
-- [ ] Windows installer
-- [ ] Improved GitHub support (platform-native UI)
-- [ ] BitBucket
-- [ ] Linux installer
+- [ ] 32-bit Windows support (only 64-bit Windows is supported currently)
+- [ ] Linux support
+- [ ] macOS/Linux native UI
 
 ## Download and Install
 
-To use GCM Core, you can download the [latest installer](https://github.com/Microsoft/Git-Credential-Manager-Core/releases/latest) for your platform. To install, double-click installation package and follow the instructions presented.
+### macOS Homebrew
 
-### Git Credential Manager for Mac and Linux (Java-based GCM)
+The preferred installation mechanism is using Homebrew; we offer a Cask in our custom Tap.
+
+To install, run the following:
+
+```shell
+brew tap microsoft/git
+brew cask install git-credential-manager-core
+```
+
+#### Git Credential Manager for Mac and Linux (Java-based GCM)
 
 If you have an existing installation of the 'Java GCM' on macOS and you have installed this using Homebrew, this installation will be unlinked (`brew unlink git-credential-manager`) when GCM Core is installed.
 
+#### Uninstall
+
+To uninstall, run the following:
+
+```shell
+brew cask uninstall git-credential-manager-core
+```
+
+---
+
+### macOS Package
+
+We also provide a [.pkg installer](https://github.com/Microsoft/Git-Credential-Manager-Core/releases/latest) with each release. To install, double-click the installation package and follow the instructions presented.
+
+#### Uninstall
+
+To uninstall, run the following:
+
+```shell
+sudo /usr/local/share/gcm-core/uninstall.sh
+```
+
+---
+
+### Windows
+
+You can download the [latest installer](https://github.com/Microsoft/Git-Credential-Manager-Core/releases/latest) for Windows. To install, double-click the installation package and follow the instructions presented.
+
+#### Git Credential Manager for Windows
+
+GCM Core installs side-by-side any existing Git Credential Manager for Windows installation and will take precendence over the older GCM. GCM Core will use any existing credentials generated by GCM for Windows so you shouldn't need to re-authenticate.
```

Comment:
"... over the older GCM."  maybe just "... over it."  There's a bit too much jargon in this paragraph.

Assuming we defined "GCM Core" up top, those 2 references are fine.

Typo: prece[N]dence

There are "Git Credential Manager for Windows" and "GCM for Windows" terms in this paragraph.
Perhaps, "... will take precedence over it and use any existing stored credentials, so you ..."

Reply:
Fixed!